File: stacks+queues/StackwArrays.py
class Stack:
    def __init__(self):
        self.array = []

    def peek(self):
        return self.array[len(self.array)-1]

    def push(self,value):
        self.array.append(value)
        return self

    def pop(self):
        self.array.pop()
        return self

myStack = Stack()
myStack.push(2)
myStack.push(10)
print(myStack.peek())
myStack.pop()
print(myStack.peek())

File: stacks+queues/StackwArrays.js
class Stack {
    constructor() {
        this.array = [];
    }

    peek() {
        return this.array[this.array.length-1];
    }

    push(value) {
        this.array.push(value);
        return this;
    }

    pop() {
        this.array.pop()
        return this
    }
}

const myStack = new Stack();
myStack.push(2);
myStack.push(10);
console.log(myStack.peek());
myStack.pop();
console.log(myStack.peek());

